website/index.html

```diff
@@ -29,24 +24,32 @@ <h2>Echo</h2>
       <pre id="output">
       </pre>
       <a href="https://github.com/rayluo/brython-project-template"><img style="position: absolute; top: 0; right: 0; border: 0; z-index:100;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png" alt="Fork me on GitHub"></a>
-      <!-- The real project content ends from here -->
-    </div>
+      <!--
+        Your main logic should go into main.py.
+        Parsing this code may cause a delay but the css animations from load.js
+        should continue to run.
 
+        It is important to remember that all text/python scripts are loaded and run
+        in order AFTER all the javascript scripts.
+        You cannot interleave python/javascript scripts and expect them to run in order.
```

Comment:
@iansparks , to address your latest comment, I move this part of "important message" into right before the inclusion of `main.py`.